src/components/banUser/BanUser.tsx

```diff
@@ -14,15 +14,41 @@ interface BanUserProps {
 	handleUserBan?: (username: string) => void;
 }
 
-export const BanUser: React.FC<BanUserProps> = ({
+interface BanUserOverlayProps {
+	overlayActive: boolean;
+	userName: string;
+	handleOverlay?: () => void;
+}
+
+export const BanUser: React.VFC<BanUserProps> = ({
 	rcUserId,
 	chatId,
 	userName,
 	handleUserBan
 }) => {
 	const { t: translate } = useTranslation();
-	const [overlayActive, setOverlayActive] = useState(false);
-	const [overlayItem, setOverlayItem] = useState<OverlayItem>();
+
+	const banUser = () => {
+		apiPostBanUser({ rcUserId, chatId }).then(() => {
+			if (handleUserBan) handleUserBan(userName);
+		});
+	};
+
+	return (
+		<>
+			<button className="banUser" onClick={banUser}>
+				{translate('banUser.ban.trigger')}
+			</button>
+		</>
```

Comment:
```suggestion
	<button className="banUser" onClick={banUser}>
		{translate('banUser.ban.trigger')}
	</button>
```

I believe there’s no need for this wrapper here :)